packages/aws-cdk-lib/region-info/build-tools/generate-static-data.ts

```diff
@@ -73,6 +74,8 @@ export async function main(): Promise<void> {
 
     registerFact(region, 'APPMESH_ECR_ACCOUNT', APPMESH_ECR_ACCOUNTS[region]);
 
+    registerFact(region, 'DEFAULT_CR_NODE_VERSION', ['aws', 'aws-cn'].includes(partition) ? Runtime.NODEJS_16_X.toString() : Runtime.NODEJS_14_X.toString()); 
```

Comment:
To make this slightly easier to manage in the future, can we do a map of `partition` -> `runtime` ?